frontend/src/utils/Copy.ts

```diff
@@ -0,0 +1,4 @@
+const TRANSLATION_PAGE_TOOL_TIP_COPY =
```

Comment:
nit: change this to `export const TRANSLATION_PAGE_TOOL_TIP_COPY ....` rather than `export default` since this file will likely grow to export alot more copy.
change the related import lines to `import { TRANSLATION_PAGE_TOOL_TIP_COPY } from "../../utils/Copy";`